tests/test_cli.py

```diff
@@ -0,0 +1,193 @@
+import pytest
+from pathlib import Path
+import subprocess
+
+import pytest
+from pathlib import Path
+import subprocess
+from lando_cli.cli import (
+    Config,
+    get_new_commits,
+    get_commit_patches,
+    get_commit_message,
+    detect_new_tags,
+    detect_merge_from_current_head,
+    get_current_branch,
+)
+
+
+@pytest.fixture
+def git_remote_repo(tmp_path: Path):
+    """Create a temporary bare remote Git repo."""
+    remote_repo = tmp_path / "remote.git"
+    subprocess.run(["git", "init", "--bare", remote_repo.as_posix()], check=True)
+    yield remote_repo
+
+
+@pytest.fixture
+def git_local_repo(tmp_path: Path, git_remote_repo: Path):
+    """Create a temporary local Git repo with remote set up."""
+    local_repo = tmp_path / "local"
+    local_repo.mkdir()
+
+    subprocess.run(["git", "init", local_repo.as_posix()], check=True, cwd=local_repo)
+    subprocess.run(["git", "config", "user.name", "Test User"], cwd=local_repo)
+    subprocess.run(["git", "config", "user.email", "test@example.com"], cwd=local_repo)
+
+    # Create an initial commit
+    (local_repo / "README.md").write_text("# Test Repo")
+    subprocess.run(["git", "add", "."], cwd=local_repo)
+    subprocess.run(["git", "commit", "-m", "Initial commit"], cwd=local_repo)
+
+    # Set up remote
+    subprocess.run(
+        ["git", "remote", "add", "origin", git_remote_repo.as_posix()], cwd=local_repo
+    )
+    subprocess.run(["git", "push", "-u", "origin", "main:main"], cwd=local_repo)
+
+    yield local_repo
+
+
+def test_config_loading(tmp_path: Path, monkeypatch):
+    config_file = tmp_path / "lando.toml"
+    config_file.write_text(
+        """
+[auth]
+api_token = "fake_token"
+user_email = "test@example.com"
+lando_url = "https://lando.test"
+"""
+    )
+    monkeypatch.setenv("LANDO_CONFIG_PATH", str(config_file))
+
+    config = Config.load_config()
+    assert config.api_token == "fake_token"
+    assert config.user_email == "test@example.com"
+    assert config.lando_url == "https://lando.test"
+
+
+def test_config_loading_default_url(tmp_path: Path, monkeypatch):
```

Comment:
nit: the `test_config_` could be a single parametrize test with a dict giving the content of the config, and something like an error message if an exception expected.